indices/tutorials.js

```diff
@@ -6,17 +6,16 @@ module.exports = new AlgoliaIndex('tutorials', getRecords())
 
 function getRecords () {
   return chain(Object.values(require('electron-i18n').docs['en-US']))
-    .filter(tutorial => !tutorial.isApiDoc && !tutorial.isApiStructureDoc)
+    .filter(tutorial => {
+      const { isApiDoc, isApiStructureDoc, slug } = tutorial
+      return !isApiDoc && !isApiStructureDoc && slug !== 'README'
```

Comment:
Note that the `Moved to` check has been removed because it seems like there are no docs with `Moved to`. We usually just do redirects on `electronjs.org` if any docs _have_ been moved around.